cmd/monolithic/development/main.go

```diff
@@ -41,6 +52,10 @@ func main() {
 	awsIoTManagerPass := flag.String("awsiot-keysecret", "", "AWS IoT Manager SecretAccessKey")
 	awsIoTManagerRegion := flag.String("awsiot-region", "eu-west-1", "AWS IoT Manager Region")
 	awsIoTManagerID := flag.String("awsiot-id", "", "AWS IoT Manager ConnectorID")
+	cryptoengineOptions := flag.String("cryptoengines", "", ", separated list of crypto engines to enable ['aws-secrets','aws-kms','vault','pkcs11','golang']")
```

Comment:
Shouldn't we add at least on default? i think this is a must. Also change "Golang" for "Filesystem" or "GolangFilesystem" to reflect the upcoming changes to crypto engines using software-generated keys with golang's crypto package